python/cudf/cudf/core/index.py

```diff
@@ -915,6 +915,21 @@ def __pos__(self):
     def __abs__(self):
         return abs(self._as_int_index())
 
+    def __dask_tokenize__(self):
```

Comment:
Right, I don't think there is a `cached`, but `lru_cache` should make sense here. (Note that I originally did this manually because I have been getting mypy warnings for `lru_cache` recently, but this doesn't seem to be an issue here)

Reply:
Ah sorry, it's `functools.cache`.